packages/auth/src/config/app.ts

```diff
@@ -42,7 +42,7 @@ export const Config = {
   identityServerSecret: envString('IDENTITY_SERVER_SECRET', 'replace-me'),
   authServerDomain: envString(
     'AUTH_SERVER_DOMAIN',
-    `http://localhost:${envInt('PORT', 3006)}`
+    `http://localhost:${envInt('AUTH_PORT', 3006)}`
```

Comment:
Will probably need to add the `breaking` label because of this?

Reply:
You are right.